ui/app/components/clients/history.js

```diff
@@ -89,48 +90,68 @@ export default class History extends Component {
     );
   }
 
-  get latestUpgradeData() {
-    // {id: '1.9.0', previousVersion: null, timestampInstalled: '2021-11-03T10:23:16Z'}
-    // version id is 1.9.0 or earliest upgrade post 1.9.0, timestamp is RFC3339
-    // need to return the last in the array as the data is in ascending order
-    return this.args.model.versionHistory.at(-1) || null;
+  get upgradeVersionHistory() {
+    const versionHistory = this.args.model.versionHistory;
+    if (!versionHistory || versionHistory.length === 0) {
+      return null;
+    }
+
+    // get upgrade data for initial upgrade to 1.9 and/or 1.10
```

Comment:
First - we only care about an upgrade to '1.9' and/or '1.10'. We are also only concerned when the _first_ upgrade to the relative minor version (subsequent patch releases, within either minor version, are irrelevant)